packages/discord.js/src/structures/AutoModerationActionExecution.js

```diff
@@ -84,6 +84,24 @@ class AutoModerationActionExecution {
   get autoModerationRule() {
     return this.guild.autoModerationRules.cache.get(this.ruleId) ?? null;
   }
+
+  /**
+   * The channel where this action was triggered from.
+   * @type {?TextBasedChannel}
+   * @readonly
+   */
+  get channel() {
+    return this.guild.channels.cache.get(this.channelId) ?? null;
+  }
+
+  /**
+   *  The the user that triggered this action.
+   * @type {?User}
+   * @readonly
+   */
+  get user() {
```

Comment:
Wouldn’t it make more sense to add a getter for a GuildMember here, since it‘s specific to a guild and if the User  is cached chances are good that so is the respective GuildMember.

Reply:
idk best to add both :)